src/Microsoft.Health.Fhir.Shared.Api/Modules/OperationsModule.cs

```diff
@@ -35,6 +36,19 @@ public void Load(IServiceCollection services)
             services.Add<ResourceToNdjsonBytesSerializer>()
                 .Singleton()
                 .AsService<IResourceToByteArraySerializer>();
+
+            services.Add<ReindexJobTask>()
+                .Transient()
+                .AsSelf();
+
+            services.Add<IReindexJobTask>(sp => sp.GetRequiredService<ReindexJobTask>())
+                .Transient()
+                .AsSelf()
+                .AsFactory();
```

Comment:
Does this work?

```suggestion
            services.Add<ReindexJobTask>()
                .Transient()
                .AsSelf()
                .AsImplementedInterfaces()
                .AsFactory<IReindexJobTask>();
```

Reply:
I followed the registration that the exportjobtask used.  But I will double check to make sure that it works.